plugins/org.eclipse.sirius.tests.swtbot/src/org/eclipse/sirius/tests/swtbot/EdgeLabelsMoveFromEdgeMoveTest.java

```diff
@@ -697,33 +697,36 @@ public void testLabelStabilityWhenMovingPointOnObliqueEdge() {
+        // ZoomLevel.ZOOM_100, "A", "B", 1, true);
 
         // Case Oblique B of the spec - The reference point is on a segment that
         // is not impacted by the move --> The label does not move.
         edgeLabelExpectedPosition.clear();
         edgeLabelExpectedPosition.put("refToBEnd", DELTA_TO_COMPUTE_FROM_RATIO);
-        doTestMovePointOnEdge(diagramName, Arrays.asList(new Point(0, 22)), edgeLabelExpectedPosition, ZoomLevel.ZOOM_100, "A", "B", 3);
-
-        // Case Oblique F of the spec - The reference point is on a segment that
-        // is removed --> The label is reset to standard location.
-        edgeLabelExpectedPosition.clear();
-        edgeLabelExpectedPosition.put("refToDCenter", DELTA_TO_COMPUTE_FROM_STANDARD);
-        edgeLabelExpectedPosition.put("refToDEnd", DELTA_TO_COMPUTE_FROM_STANDARD);
-        doTestMovePointOnEdge(diagramName, Arrays.asList(new Point(0, -123)), edgeLabelExpectedPosition, ZoomLevel.ZOOM_100, "C", "D", 2, true);
-
-        // Case Oblique G of the spec - The reference point is on a segment that
-        // is splitted into two segments --> The label is reset to standard
-        // location.
-        edgeLabelExpectedPosition.clear();
-        edgeLabelExpectedPosition.put("refToBEnd", DELTA_TO_COMPUTE_FROM_STANDARD);
-        edgeLabelExpectedPosition.put("refToBBegin", new Dimension(0, 0));
-        edgeLabelExpectedPosition.put("refToBCenter", new Dimension(0, 0));
-        doTestMoveSegment(DIAGRAM_DESCRIPTION_NAME, diagramName, Arrays.asList(new Point(0, -100)), edgeLabelExpectedPosition, ZoomLevel.ZOOM_100, "A", "B", 2, false);
+        doTestMovePointOnEdge(diagramName, Arrays.asList(new Point(0, 22)), edgeLabelExpectedPosition, ZoomLevel.ZOOM_100, "A", "B", 3, true);
+
+        // // Case Oblique F of the spec - The reference point is on a segment that
+        // // is removed --> The label is reset to standard location.
+        // edgeLabelExpectedPosition.clear();
+        // edgeLabelExpectedPosition.put("refToDCenter", DELTA_TO_COMPUTE_FROM_STANDARD);
+        // edgeLabelExpectedPosition.put("refToDEnd", DELTA_TO_COMPUTE_FROM_STANDARD);
+        // doTestMovePointOnEdge(diagramName, Arrays.asList(new Point(0, -123)), edgeLabelExpectedPosition,
+        // ZoomLevel.ZOOM_100, "C", "D", 2, true);
+        //
+        // // Case Oblique G of the spec - The reference point is on a segment that
+        // // is splitted into two segments --> The label is reset to standard
+        // // location.
+        // edgeLabelExpectedPosition.clear();
+        // edgeLabelExpectedPosition.put("refToBEnd", DELTA_TO_COMPUTE_FROM_STANDARD);
+        // edgeLabelExpectedPosition.put("refToBBegin", new Dimension(1, 0));
+        // edgeLabelExpectedPosition.put("refToBCenter", new Dimension(0, 0));
+        // doTestMoveSegment(DIAGRAM_DESCRIPTION_NAME, diagramName, Arrays.asList(new Point(0, -100)),
+        // edgeLabelExpectedPosition, ZoomLevel.ZOOM_100, "A", "B", 2, false);
```

Comment:
why is it commented?

Reply:
It was a mistake. I have not removed the temporary comments to debug the problem.